pkg/instrumentation/podmutator.go

```diff
@@ -232,7 +234,7 @@ func (pm *instPodMutator) Mutate(ctx context.Context, ns corev1.Namespace, pod c
 		pm.Recorder.Event(pod.DeepCopy(), "Warning", "InstrumentationRequestRejected", "support for Java auto instrumentation is not enabled")
 	}
 
-	if inst, err = pm.getInstrumentationInstance(ctx, ns, pod, annotationInjectNodeJS); err != nil {
+	if inst, err = pm.getInstrumentationInstance(ctx, ns, pod, annotationInjectNodeJS, make(map[string]string)); err != nil {
```

Comment:
```suggestion
	if inst, err = pm.getInstrumentationInstance(ctx, ns, pod, annotationInjectNodeJS, nil); err != nil {
```

Reply:
Again I do not like nil since it could lead to panic. Empty map is safer.

Reply:
done